File: 3. CursoPOO/Python/uberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMateral = []

    def __init__(self, license, driver, typeCarAccepted, seatsMateral):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMateral = seatsMateral

File: 3. CursoPOO/Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMateral;

    public UberBlack(String license, Account driver, Map<String, Map<String,Integer>> typeCarAccepted, ArrayList<String> seatsMateral) {
        // super hace referencia a los atributos de la clase padre
        super(license, driver);
        // this hace referencia a los atributos de la clase hija
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMateral = seatsMateral;
    

    }

}
